src/healthcareapis/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.2.1
```

Comment:
I wonder if the version should be 0.3.0.  Typically, the last number denotes minor bug fixes.  In this case, we are adding a new features/capabilities.

Reply:
I agree, fixed and all other instances in healthcareapis